secure_fw/partitions/internal_trusted_storage/flash_fs/its_flash_fs.h

```diff
@@ -167,15 +167,23 @@ struct its_flash_fs_ops_t {
 typedef struct its_flash_fs_ctx_t its_flash_fs_ctx_t;
 
 /*!
- * \struct its_file_info_t
+ * \struct its_flash_fs_file_info_t
  *
- * \brief Structure to store the file information.
+ * \brief Structure containing file information.
+ *
+ * \details This structure is not written to the filesystem, it is used by the
+ *          file system functions to simplify accessing the containing
+ *          information.
  */
-struct its_file_info_t {
-    size_t size_current; /*!< The current size of the flash file data */
-    size_t size_max;     /*!< The maximum size of the flash file data in bytes.
-                          */
-    uint32_t flags;      /*!< Flags set when the file was created */
+struct its_flash_fs_file_info_t {
+    size_t size_current;  /*!< The current size of the file in bytes */
+    size_t size_max;      /*!< The maximum size of the file in bytes. */
+    uint32_t flags;       /*!< Flags set when the file was created */
+#ifdef TFM_ITS_ENCRYPTED
+    uint8_t add[ITS_FILE_ID_SIZE + 8];     /*!< Additional authenticated data */
```

Comment:
Could we improve the +8 here? Maybe add a define or something or an explanation what is this +8?

Reply:
I think that it is more clear to add a define where we say that the additional data are the ITS_FILE_SIZE_ID_SIZE + data size which is 4 bytes + flags which is 4 bytes as well.

Reply:
Ahh I din't know myself what it is... 
Added two defines for it